src/main/java/uk/org/openbanking/datamodel/payment/OBDomestic1.java

```diff
@@ -44,7 +44,7 @@
   private String localInstrument = null;
 
   @JsonProperty("InstructedAmount")
-  private OBDomestic1InstructedAmount instructedAmount = null;
```

Comment:
Can you not delete OBDomestic1InstructedAmount? (same comment for the other classes)

Reply:
I have deleted it https://github.com/ForgeRock/OpenBanking-Java-SDK/pull/21/files#diff-eb4fee6d34133203c7ea2d85a68ac834

Reply:
ah yea, my bad. not use to github review yet